### ISSUE TITLE
Flutter 3 support (Fix Here)

### ISSUE THREAD

**Issue by masreplay** [jonataslaw/readmore#50]
```yaml
readmore:
    git:
      url: https://github.com/masreplay/readmore.git
```

**Comment by andes2912** [comment]
> ```yaml
> readmore:
>     git:
>       url: https://github.com/masreplay/readmore.git
> ```

Support for html tag ?

**Comment by jonataslaw** [comment]
Fixed in [3.0.0](https://github.com/jonataslaw/readmore/releases/tag/3.0.0)
